app/views/unattended/provisioning_templates/provision/autoyast_sles_default.erb

```diff
@@ -242,6 +242,22 @@ rm /etc/resolv.conf
       </script>
     </chroot-scripts>
   </scripts>
+<% if os_major >= 15 -%>
```

Comment:
Is the configuration of the firewall necessary? What would happen, if there is no configuration of the firewall?

Reply:
Starting with SP4 the defaults changed leading to the SSH port being closed after the OS installation.
I think instead of relying on defaults it is better to explicitly configure the Firewall.

Reply:
ok. make sense.